client/src/project/Project.state.js

```diff
@@ -246,15 +246,14 @@ class ProjectModel extends StateModel {
     if (this.get("transient.requests.filesTree") === SpecialPropVal.UPDATING) return;
     const oldTree = this.get("filesTree");
     openFilePath = this.cleanFilePathUrl(openFilePath);
-    if (isNullOrUndefined(oldTree))
+    if (oldTree === null || oldTree === undefined)
```

Comment:
This can be simplified as
```suggestion
    if (oldTree == null)
```